mocha-client-tests/addon/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Mocha tests",
+  "version": "1.0",
+  "manifest_version": 2,
+  "description": "Check ",
+  "icons": {
+    "16": "images/icon-16.png"
+  },
+  "short_name": "MochaTest",
+  "background": {
+    "scripts": [
+      "background.js"
+    ]
+  },
+  "permissions": [
+    "activeTab",
+    "tabs",
+    "http://*/*",
+    "https://*/*",
+    "bookmarks",
+    "notifications",
+    "storage",
+    "<all_urls>"
+  ],
+  "content_scripts": [
+    {
+      "matches": [
+        "http://*/*",
+        "https://*/*"
+      ],
+      "js": [
+        "bower_components/jquery/dist/jquery.min.js",
+        "contentscript.js"
+      ],
+      "run_at": "document_end",
+      "all_frames": false
+    }
+  ],
+  "browser_action": {
+    "default_icon": {
+      "19": "images/icon-19.png"
+    },
+    "default_title": "Mocha Test",
+    "default_popup": "popup.html"
+  },
+  "applications": {
+    "gecko": {
+      "id": "production@gemshelf.com",
```

Comment:
We've been trying to remove the id out of example extensions, they can be through temporary install locally or pushed to AMO for signing. If there's no id, they won't all conflict with each other.